cookbooks/ey-mysql/recipes/install.rb

```diff
@@ -49,6 +49,8 @@
 end
 
 package "libmysqlclient-dev"
+# installs mysql client to all instnaces. 
+package "percona-server-client"
```

Comment:
@rupikakapoor is this package available on ARM instances? And if not, which package should be installed?

Reply:
https://github.com/engineyard/ey-cookbooks-stable-v7/pull/197/files#diff-fe00c2963ae2cc2f984fad392f1b3a218a432afde43c2905d8c81f66ea3c82d6

Reply:
@mushyy @dvalfre Added condition to handle installation of `percona-server-client` for amd64 and for arm64 `mysql-client`